client/src/app/site/pages/meetings/pages/participants/pages/participant-detail/pages/participant-detail-manage/components/participant-create-wizard/participant-create-wizard.component.ts

```diff
@@ -362,4 +363,12 @@ export class ParticipantCreateWizardComponent extends BaseMeetingComponent imple
             .map((id: Id): ViewGroup => this.groupRepo.getViewModel(id))
             .some(group => group.hasPermission(Permission.userCanManage));
     }
+
+    private cleanUpGroups(group_ids: number[]): number[] {
+        const meeting_group_ids = this.activeMeetingService.meeting.group_ids;
+        group_ids = group_ids.filter(group_id_1 =>
+            meeting_group_ids.find(group_id_2 => group_id_1 === group_id_2)
+        );
```

Comment:
Why not use `group_id => meeting_group_ids.includes(group_id)`  as filter function?
That would be a bit clearer.

Else Okay, please test.

Reply:
One other objection of this code is, that it clears symptoms and not the cause of the bug. I am fine with that, but I think @bastianjoel won't like it.

Reply:
@reiterl You review this. If you think there is a better way to do this request a change.

Reply:
Okay, I think, that the `includes` use is better.